rmf_demos/launch/simulation.launch.xml

```diff
@@ -25,6 +26,9 @@
       <env name="GAZEBO_MODEL_DATABASE_URI" value="" />
       <env name="MENGE_RESOURCE_PATH" value="$(var menge_resource_path)"/>
     </executable>
+    <executable cmd="bash -c &quot;sleep 10; gz physics -u $GAZEBO_REAL_TIME_UPDATE_RATE&quot;" output="both">
```

Comment:
Could you elaborate on why there's a `sleep 10;` here?

Reply:
if I run this command without sleep, my gazebo instance seems to hang. I'm not entirely sure why, but adding a sleep seems to handle this. This is the "hacky" part of this solution. Perhaps there is a better way of dealing with this, I'd love to pick your brain on this.